test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md

```diff
@@ -147,22 +147,33 @@ oc get rhmi rhoam -n redhat-rhoam-operator -o json | jq -r .status.quota
 
 **Verify that LDAP IDP can be configured**
 
-1. Go to [AWS secrets file in 'vault' repository](https://gitlab.cee.redhat.com/integreatly-qe/vault/-/blob/master/SECRETS.md) again and search for "ldap-server-tester AWS UI credentials"
+1. Go to [AWS secrets file in 'vault' repository](https://gitlab.cee.redhat.com/integreatly-qe/vault/-/blob/master/SECRETS.md) again and search for "RHOAM LDAP Instance" You should find "Login page" there and AWS credentials for AWS CLI
 2. Access the AWS EC2 console with those credentials
 
+   - you can either use the IAM user if you have one and navigate to "Login page" in your browser
+   - or you can use AWS CLI
+
 3. On the AWS EC2 console, right click on the LDAP Server EC2 instance and select start instance.
 
    > Instance should change to running state
 
+   If having only AWS CLI access you can do the following
+
+   ```bash
```

Comment:
to prevent opening AWS pager
```suggestion
   ```bash
   export AWS_PAGER=""
```

Reply:
Accepted, thanks